source/development/coredump.rst

```diff
@@ -117,3 +117,46 @@ On macOS, most applications have core dump generation disabled by default. Howev
       # sysctl -w kern.corefile=/cores/core.%P
 
 Enabling core dump generation might consume significant disk space, so use it judiciously. Moreover, not all processes on macOS support or behave consistently with core dump generation.
+
+Windows endpoints
+-----------------
+
+To collect user-mode dumps on Windows, you can use the built-in Windows Error Reporting (WER) feature. Here's how you can set it up:
+
+#. Press **Windows + R** keys on your keyboard to open the run dialog box.
+
+#. Type **regedit** in the search box and click **OK** to open the registry editor.
+
+#. Navigate to the Registry Key:
+
+   .. code-block:: none
+
+        HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\Windows Error Reporting\LocalDumps
+
+   .. note:: If the ``LocalDumps`` key doesn't exist, you may need to create it.
+
+#. Press **Right-click** on the LocalDumps key and choose **New > Key**
+
+    * Name the new key how to ``wazuh-agent.exe``
+
+   .. note:: Inside the newly created key, you will have to set some property values to configure dump collection
+
+#. Press **Right-click** inside the LocalDumps key and choose **New > Expandable String Value**
+    
+    * Name the new key how to ``DumpFolder``
+    * Press **Right-click** on the DumpFolder property and choose **Modify**
+    * Change the added string value to ``%LOCALAPPDATA%\WazuhCrashDumps``
+
+#. Press **Right-click** inside the LocalDumps key and choose **New > DWORD (32-bit) Value**
+
+    * Name the new key how to ``DumpType``
+    * Press **Right-click** on the DumpType property and choose **Modify**
+    * Change the added number value to ``2``
+
+#. Close the regedit tool and restart the Wazuh agent via PowerShell with administrator privileges:
+
+   .. code-block:: PowerShell
+
+      > Restart-Service -Name wazuh
+
+Remember to be cautious when modifying the Windows Registry. Incorrect changes can cause system instability or other issues. Always backup the registry or create a system restore point before making changes.
```

Comment:
I would place this warning at first, before any modifications have been advised. Maybe highlighted with a note frame

Reply:
done